package.json

```diff
@@ -55,6 +55,11 @@
           "description":
           "Override the default path of the erlang_ls executable with a custom one."
         },
+        "erlang_ls.config.path": {
```

Comment:
All the other names for the configuration values are using camelCase for the names, shouldn't we keep it that way for this configuration as well?